xtask/src/main.rs

```diff
@@ -23,24 +25,38 @@ pub struct Options {
 
 #[derive(Debug, Parser)]
 enum Command {
+    #[cfg(target_os = "linux")]
     BuildEbpf(build_ebpf::Options),
-    BuildProto(build_proto::Options),
-    Run(run::Options),
+    #[cfg(target_os = "linux")]
     GrpcClient(grpc::Options),
```

Comment:
So wait, wouldn't we want the grpc client build available on other OS?

Reply:
this disables anything on non-linux that imported any dataplane crates. upon further thinking, now that we have the bindings generated for the controlplane crate as well, we can use that crate to import the bindings in the grpc client tool.